styx-cli/src/main/java/com/spotify/styx/cli/Main.java

```diff
@@ -363,7 +365,9 @@ private String apiUrl(CharSequence... parts) {
   }
 
   private ByteString send(Request request) throws ExecutionException, InterruptedException {
-    final Response<ByteString> response = client.send(request).toCompletableFuture().get();
+    final Response<ByteString> response =
+        client.send(request.withHeader("Client-Version", STYX_CLI_VERSION)).toCompletableFuture()
```

Comment:
Would be better to use ```User-Agent``` indeed. I wasn't sure how Apollo will deal with this header, hopefully not rewriting it. I will do some experiment or read doc.